utils/cq/daos_pylint.py

```diff
@@ -235,189 +240,201 @@ def run(self, args):
         print(self)
         failed = False
         if self.files:
-            if parse_file(args, self.files):
+            if self.parse_file(args, self.files):
                 failed = True
         if self.ftest_files:
-            if parse_file(args, self.ftest_files, ftest=True):
+            if self.parse_file(args, self.ftest_files, ftest=True):
                 failed = True
         if self.fake_scons:
-            if parse_file(args, self.fake_scons, fake_scons=True):
+            if self.parse_file(args, self.fake_scons, fake_scons=True):
                 failed = True
         if self.scons_files:
             for file in self.scons_files:
-                if parse_file(args, file, scons=True):
+                if self.parse_file(args, file, scons=True):
                     failed = True
         return failed
 
+    def parse_file(self, args, target_file, ftest=False, scons=False, fake_scons=False):
+        """Parse a list of targets.
+
+        Returns True if warnings issued to GitHub."""
```

Comment:
FYI - In ftest at least, we follow this format
```suggestion
        """Parse a list of targets.

        Args:
            args (<type>): <desc>
            ...

        Returns:
            bool: True if warnings issued to GitHub.
        """
```